api/modules/feedback/views.py

```diff
@@ -30,3 +31,44 @@ def add_feedback(request):
 
     success_message = "Sucessfully added new feedback."
     return Response(success_message, status=status.HTTP_201_CREATED)
+
+
+@api_view(['GET'])
+def get_feedback(request, feedback_id):
+    """
+    Returns the feedback pertaining to a certain feedback id
+    :param request:
+    :return: 400 if incorrect parameters are sent or database request failed
+    :return: 201 successful
+    """
+    try:
+        user_feedback = Feedback.objects.get(pk=feedback_id)
+
+    except Feedback.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+
+    serializer = FeedbackCondensedSerializer(user_feedback)
+    return Response(serializer.data)
+
+
+""" The many specified here is important because we have a one to many relation and
+has to be specified in both the serializer as well as here. """
+
+
+@api_view(['GET'])
+def get_all_user_feedback(request):
+    """
+    Returns a list of all the feedbacks for a given user
+    :param request:
+    :return: 200 successful
+    """
+    try:
+        feedbacks = Feedback.objects.get(user=request.user)
+        if request.user not in feedbacks.user.all():
```

Comment:
no need for this check since you will be getting only current user's feedbacks
So remove this if block